lms/djangoapps/instructor/views/api.py

```diff
@@ -2334,7 +2335,21 @@ def list_instructor_tasks(request, course_id):
         - `problem_location_str` and `unique_student_identifier` lists task
             history for problem AND student (intersection)
     """
-    return _list_instructor_tasks(request=request, course_id=course_id)
+    permission_classes = (IsAuthenticated, permissions.InstructorPermission)
+    permission_name = permissions.SHOW_TASKS
+    serializer_class = ListInstructorSerializer
+
+    @method_decorator(ensure_csrf_cookie)
+    def post(self, request, course_id):
```

Comment:
It looks like this API was previously a POST method but with query string input params,  Can we just update the instructor dash to call this as a `GET` method instead since it doesn't actually modify any data?

Reply:
They are not sending any data. Its a just a post call without any payload. I just maintained the behaviour. May be some other front-end is using it this way ? 

Also another similar [api](https://github.com/openedx/edx-platform/blob/275d4d989fd0a6ba5033448eca3228c42d572a42/lms/djangoapps/instructor/views/api_urls.py#L17) exists with `GET` and its hitting same method for getting data. But its different version.

Reply:
Interesting, okay seems like we have some redundancy that we should cleanup but that can be part of some future work.  I don't want to blow up the scope of this change.